ui/components/GrafanaCustomChart.js

```diff
@@ -44,7 +44,8 @@ const grafanaStyles = (theme) => ({
   },
   card: {
     height: '100%',
-    width: "100%"
+    width: "100%",
+    background:'#f3efff',
```

Comment:
![image](https://user-images.githubusercontent.com/44001539/124842737-e89ef580-dfad-11eb-94d6-67d624173c35.png)

Are we sure, we want to go with purple?

Light green/blue might be bettor option considering the Brand Guide.

I tried out few colors that might be better aligned with the style.
(#eaf8f5)
![image](https://user-images.githubusercontent.com/44001539/124843007-98746300-dfae-11eb-9788-54dd8a941104.png)

or (#eaf6fc)
![image](https://user-images.githubusercontent.com/44001539/124843157-ee490b00-dfae-11eb-9368-9d1463b286a2.png)

What do you think, @Ayush-Khandelwal-007 ?